tests/database/factories/ContractorFactory.php

```diff
@@ -7,6 +7,6 @@
 
 $factory->define(Contractor::class, function (Faker $faker): array {
     return [
-        'position' => $faker->jobTitle,
+        'position' => $faker->word,
```

Comment:
Changed because of method not being available in old laravel versions

Reply:
Fine.